scripts/run-encryption-tests-local.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+# sets up an encrypted mongodb cluster, adds relevant variables to the environment, and runs encryption tests
```

Comment:
(this applies everywhere throughout this PR) there isn't a such thing as an "encrypted cluster".  clusters that use FLE are still just regular mongod/mongoses, but there are some additional setup steps we need to do to use FLE.  We must
 
1. have mongocryptd or the shared library downloaded
2. have an enterprise server >= 4.2

Additionally, I'm pretty sure that as the server changes, the shared library does too - so we should use the same crypt_shared as was included in the server download.

--

drivers-evergreen-tools handles all of this for us - so all we need to do is run the bash script and we get:

- an enterprise server running with the configuration that we chose (enterprise by default, I think you can change that)
- mongocryptd downloaded
- crypt_shared downloaded

We could use other setup tools, but it's just simplest to use drivers-evergreen-tools to set up the cluster because that's what we use.

Could we update the language here to be a bit more precise?  For example, instead of saying "encrypted cluster", perhaps we can say "downloads all tools required to use FLE with the driver / mongoose and and starts a cluster of the provided configuration".

Reply:
That's a good point - I've pushed up the other changes you've requested as well. 


Tracking note: Bailey's on leave starting last Friday and I was directed to focus on client.close() work so I didn't push up the small request changes until now.